internal/verifier/verifier.go

```diff
@@ -70,14 +61,23 @@ const (
 
 // Verifier is the object that verifies artifacts
 type Verifier struct {
-	verifier ArtifactVerifier
+	verifier verifyif.ArtifactVerifier
```

Comment:
Do you want to use a type assertion to document that Verifier implements ArtifactVerifier?

```golang
var _ verifyif.ArtifactVerifier= (*Verifier)(nil)
```

Reply:
sure!